BDArmory/UI/BDArmoryAIGUI.cs

```diff
@@ -342,7 +342,7 @@ void SetInputFields(Type AIType)
                         { "autoTuningAltitude", gameObject.AddComponent<NumericInputField>().Initialise(0, ActivePilot.autoTuningAltitude, 50, 5000) },
                         { "autoTuningSpeed", gameObject.AddComponent<NumericInputField>().Initialise(0, ActivePilot.autoTuningSpeed, 50, 800) },
 
-                        { "defaultAltitude", gameObject.AddComponent<NumericInputField>().Initialise(0, ActivePilot.defaultAltitude, 100, 15000) },
+                        { "defaultAltitude", gameObject.AddComponent<NumericInputField>().Initialise(0, ActivePilot.defaultAltitude, 50, 15000) },
```

Comment:
Many of these values should actually depend on the slider resolution setting. It's been on my to-do list for a while to have these get adjusted to use the right values when the slider changes.